ksp_plugin/plugin.cpp

```diff
@@ -473,14 +473,14 @@ FrameField<World> Plugin::Navball(
           sun_world_position,
           sun_->current_position(current_time_),
           to_world.Inverse());
-  return [rendering_frame, to_world, positions_from_world, this](
+  return [navigation_frame, to_world, positions_from_world, this](
```

Comment:
Preference for capturing `this` first, and then the other parameters in alphabetical order.